packages/fxa-content-server/app/scripts/templates/reset_password.mustache

```diff
@@ -1,12 +1,15 @@
 <div id="main-content" class="card">
   <header>
     <h1 id="fxa-reset-password-header">
+      {{#t}}Reset password{{/t}}
```

Comment:
I really wish I'd have extracted this header stuff into a partial template so that this would only need to be updated in one place. :(

Reply:
Yeah, I noticed some redundancy as I was working through this. Perhaps it can be tackled with the settings redesign.